src/styles/_catalog.scss

```
@@ -14,9 +14,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```

```suggestion
    transition: color 300ms;
```
Краще завжди вказувати, до чого саме буде застосовано transition